### ISSUE TITLE
Update Ruby version 3.0.0

### ISSUE THREAD

**Issue by IjayAbby** [microverseinc/linters-config#123]
Hello, currently we have Ruby version 3.0.0 but this linters do not support the latest version.
Kindly update it to make it easier for configuration with the latest Ruby version.
Thanks.

**Comment by bolah2009** [comment]
@euqueme @uryelah Can you please check this issue?

**Comment by uryelah** [comment]
> 
> 
> @euqueme @uryelah Can you please check this issue?

We could try a [matrix strategy](https://docs.github.com/en/actions/guides/building-and-testing-ruby#testing-with-multiple-versions-of-ruby) to accept multiple versions of ruby. I'll test this out.

```
name: Ruby CI

on:
  push:
    branches: [ $default-branch ]
  pull_request:
    branches: [ $default-branch ]

jobs:
  test:

    runs-on: ubuntu-latest

    strategy:
      matrix:
        ruby-version: [2.7.x, 2.6.x, 2.5.x]

    steps:
    - uses: actions/checkout@v2
    - name: Set up Ruby ${{ matrix.ruby-version }}
      uses: ruby/setup-ruby@v1
      with:
        ruby-version: ${{ matrix.ruby-version }}
    - name: Install dependencies
      run: bundle install
    - name: Run tests
      run: bundle exec rake

```

**Comment by uryelah** [comment]
@IjayAbby Coudl you send a screenshot of the error? 
Did you have a `.ruby-version` file in the project?
Did both actions fail or just one of them, Rubocop or Rspec.

**Comment by uryelah** [comment]
It seems that Rubocop runs ok on projects built with ruby version 3 unless there's a `.ruby-version` file

**Comment by bolah2009** [comment]
@uryelah if we update the current version of robocop, would that solve this issue?

**Comment by uryelah** [comment]
@bolah2009 I'm double checking this but I really think just asking students to not commit .ruby-version files would be enough.

**Comment by uryelah** [comment]
We don't have a specifies rubocop version, only an specified ruby ver:
```
        with:
          ruby-version: 2.6.x
```

**Comment by uryelah** [comment]
@bolah2009 
We could change the rubocop version to 1.9 and accept any ruby version above 2.6.x
What do you think? That seems to work properly.

Ex:
```
name: Linters

on: pull_request

jobs:
  rubocop:
    name: Rubocop
    runs-on: ubuntu-18.04
    
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-ruby@v1
        with:
          ruby-version: ">=2.6.x"
      - name: Setup Rubocop
        run: |
          gem install --no-document rubocop:'~>1.9.0' # https://docs.rubocop.org/en/stable/installation/
          [ -f .rubocop.yml ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ruby/.rubocop.yml
      - name: Rubocop Report
        run: rubocop --color
```

**Comment by uryelah** [comment]
@bolah2009 [PR](https://github.com/microverseinc/linters-config/pull/126) #126  created to add fix for 3.0.0 compatibility.
Please check this when you can.
CC @euqueme

**Comment by bolah2009** [comment]
Merged in https://github.com/microverseinc/linters-config/pull/126
